lib/charms/prometheus_k8s/v0/prometheus.py

```diff
@@ -317,7 +317,7 @@ def _on_scrape_targets_changed(self, event):
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 4
+LIBPATCH = 5
```

Comment:
Not required since we have not yet published 4. If we push this to 5 then 4 will be missing from version history of charm library.